content/en/continuous_integration/intelligent_test_runner/setup/javascript.md

```diff
@@ -19,33 +19,30 @@ further_reading:
 
 Intelligent Test Runner for JavaScript skips entire _test suites_ (test files) rather than individual tests.
 
-### Inconsistent test durations
-
-In some frameworks, such as `jest`, there are cache mechanisms that make tests faster after other tests have run (see [jest cache][4] docs). If Intelligent Test Runner is skipping all but a few test files, these suites might run slower than they usually do. This is because they run with a colder cache. Regardless of this, total execution time for your test command should still be reduced.
 
 ## Compatibility
 
 Intelligent Test Runner is only supported in the following versions and testing frameworks:
 
 * `jest>=24.8.0`
-  * From `dd-trace>=3.16.0` or `dd-trace>=2.29.0`.
+  * From `dd-trace>=4.17.0` or `dd-trace>=3.38.0`.
   * Only `jest-circus/runner` is supported as `testRunner`.
   * Only `jsdom` and `node` are supported as test environments.
 * `mocha>=5.2.0`
-  * From `dd-trace>=3.16.0` or `dd-trace>=2.29.0`.
-  * Run mocha with [`nyc`][1] to enable [code coverage][5].
+  * From `dd-trace>=4.17.0` or `dd-trace>=3.38.0`.
+  * Run mocha with [`nyc`][1] to enable code coverage.
 * `cucumber-js>=7.0.0`
-  * From `dd-trace>=3.16.0` or `dd-trace>=2.29.0`.
-  * Run cucumber-js with [`nyc`][1] to enable [code coverage][5].
+  * From `dd-trace>=4.17.0` or `dd-trace>=3.38.0`.
+  * Run cucumber-js with [`nyc`][1] to enable code coverage.
 * `cypress>=6.7.0`
-  * From `dd-trace>=4.2.0`, `dd-trace>=3.23.0` or `dd-trace>=2.36.0`.
-  * Instrument your web application with [code coverage](#cypress)
+  * From `dd-trace>=4.17.0` or `dd-trace>=3.38.0`.
```

Comment:
it's not great that we keep increasing the minimum supported versions, but I'd say specifically unskippable tests is pretty important